File: 프로그래머스/lv0/120811. 중앙값 구하기/중앙값 구하기.java
import java.util.Arrays;

class Solution {
    public int solution(int[] array) {
        int answer = 0;
        Arrays.sort(array);
        // 일단 배열 array를 오름차순으로 정렬해주고
        for(int i = 0; i < array.length; i++){
            if(array[i] == array[array.length / 2]){
                // array배열길이를 2로 나눈 몫은 딱 중간값을 나타내므로
                // array배열의 i번째 값이 중간값과 같다면 answer에 값을 넣어준다.
                answer = array[i];
            }
            
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120811. 중앙값 구하기/중앙값 구하기.js
function solution(array) {
    array.sort((a,b) => a > b ? 1 : -1); // 오름차순으로 정렬하고 
    return array[Math.floor(array.length/2)]  // 중간값을 구하기 위해 길이를 2로 나누고 실수는 버린다.
}